index.js

```diff
@@ -11,3 +12,116 @@ document.querySelectorAll('.nav-link').forEach((n) => n.addEventListener('click'
+    liveVersion: 'https://ouail-01.github.io/Portfolio_project/',
+    sourceCode: 'https://github.com/Ouail-01',
+  },
+
+  {
+    title: 'Keeping track of hundreds of components',
+    description: `Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the
+    industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and
+    scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting
+    industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown
+    printer took a galley of type and scrambled it 1960s with the relea`,
+    featuredImage: 'img/imgHolder2.png',
+    technology: ['CodeKit', 'GitHub', 'javascript', 'Bootstrap', 'Terminal', 'Codepen'],
+    liveVersion: 'https://ouail-01.github.io/Portfolio_project/',
+    sourceCode: 'https://github.com/Ouail-01',
+  },
+
+  {
+    title: 'Keeping track of hundreds of components',
+    description: `Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the
+    industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and
+    scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting
+    industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown
+    printer took a galley of type and scrambled it 1960s with the relea`,
+    featuredImage: 'img/imgHolder2.png',
+    technology: ['Ruby on rails', 'Css', 'javascript', 'Bootstrap', 'Terminal', 'GitHub'],
+    liveVersion: 'https://ouail-01.github.io/Portfolio_project/',
+    sourceCode: 'https://github.com/Ouail-01',
+  },
+
+];
+// const cardsButton = document.querySelectorAll('.project-button');
+
+cardsButton.forEach((item, i) => item.addEventListener('click', () => {
+  popupWindow.classList.toggle('active');
+  const project = myProjects[i];
+  const popContainer = document.querySelector('.popup-window');
+
+  const popupTitle = popContainer.querySelector('.popup-title');
+  popupTitle.textContent = project.title;
+  const popupDescription = popContainer.querySelector('.popup-description');
+  popupDescription.textContent = project.description;
+  const popupImage = popContainer.querySelector('.popup-image');
+  popupImage.src = project.featuredImage;
+  const popupThechnology = popContainer.querySelectorAll('.popup-technology');
+  popupThechnology.forEach((item, i) => {
+    item.textContent = project.technology[i];
+  });
+  const popupLiveVersion = popContainer.querySelector('.popup-button-live');
+  popupLiveVersion.href = project.liveVersion;
+  const popupSourceCode = popContainer.querySelector('.popup-button-source');
+  popupSourceCode.href = project.sourceCode;
+}));
+
+burgerPopup.addEventListener('click', () => {
+  popupWindow.classList.remove('active');
+});
```

Comment:
- [ ] Kindly make sure the button to close the model is visible on your page as you may notice here on your page, when the popup is active there's no way to make it disappear. kindly fix that.
- [ ] Also make sure you stick to the design as the Figma template provided. as you may notice your buttons are not quite well positioned as is the case in Figma and the project description doesn't have enough space.
`Hint: You may consider giving your modal enough height and adding some scroll effect on it or you may consider reducing the size of your image`

![image](https://user-images.githubusercontent.com/97100091/208942372-94f9b99e-176c-4a26-ad9a-0a34f68cf5b4.png)

Figma template
![image](https://user-images.githubusercontent.com/97100091/208942902-f201d5c9-d7ce-47bd-a791-94c952316351.png)

Reply:
- [ ] Even on mobile kindly make sure the user of your page will have a great experience and will be able to see all the content and interact with the page. `the close button must be visible`
Your page :point_down: 
![image](https://user-images.githubusercontent.com/97100091/208944350-894fe883-4b93-441a-a697-3881c46ee72b.png)